README.md

```diff
@@ -1,50 +1,57 @@
-[TiKV]: https://github.com/tikv/tikv
+> Each course is developed independently, so they vary in their presentation and their expectations from course-takers. Please see the individual course documentation for details.
 
+## Our Courses
 
-## Training courses
+### Series 1: Open Source Collaboration
 
-- **[Practical Networked Applications in Rust][rust]**. A series of projects
-  that incrementally develop a single Rust project from the ground up into a
-  high-performance, networked, parallel and asynchronous key/value store. Along
-  the way various real-world and practical Rust development subject matter are
-  explored and discussed.
+Open source collaboration includes a series of open source related learning mateials to help open source enthusiasts have a basic knowledge of what open source software is, the differences among kinds of open source software licenses, how to participate in open source projects and what a welcoming open source community looks like. Courses of this series are:
 
-- **[Distributed Systems in Rust][dss]**. Adapted from the [MIT 6.824]
-  distributed systems coursework, this course focuses on implementing important
-  distributed algorithms, including the [Raft] consensus algorithm, and the
-  [Percolator] distributed transaction protocol.
+- [TP 101: Introduction to open source software](courses/tp101-intro-to-oss.md)
+- [TP 102: How to use Git and GitHub](courses/tp102-how-to-use-git-github.md)
+- [TP 103: Build a welcoming community](courses/tp103-open-source-community.md)
 
-- **[Distributed Systems in Go][go]**. A course on implementing algorithms
-  necessary in distributed databases, including map reduce, and parallel query
-  optimization.
+### Series 2: Rust Programming
 
-[rust]: ./rust/
-[dss]: ./dss/
-[go]: ./tidb/
+Two courses are included in this series, which are:
 
-[MIT 6.824]: http://nil.csail.mit.edu/6.824/2017/index.html
-[Raft]: https://raft.github.io/
-[Percolator]: https://storage.googleapis.com/pub-tools-public-publication-data/pdf/36726.pdf
+- [TP 201: Practical Networked Applications in Rust](rust/README.md). A series of projects that incrementally develop a single Rust project from the ground up into a high-performance, networked, parallel and asynchronous key/value store. Along the way various real-world and practical Rust development subject matter are explored and discussed.
+
+- [TP 202: Distributed Systems in Rust](dss/README.md). Adapted from the [MIT 6.824](http://nil.csail.mit.edu/6.824/2017/index.html) distributed systems coursework, this course focuses on implementing important distributed algorithms, including the [Raft](https://raft.github.io/) consensus algorithm, and the [Percolator](https://storage.googleapis.com/pub-tools-public-publication-data/pdf/36726.pdf) distributed transaction protocol.
+
+### Series 3: Distributed Database 
+
+Two courses are included in this series, which are:
+
+- TP 301: TinyKV, a distributed key value database in Go(WIP) 
```

Comment:
should we link them to the repository of `tinysql` and `tinykv`?

Reply:
It seems to work, what do you think? @Connor1996 @winoros

Reply:
so, we do not need to move it into talent-plan repo?

Reply:
yes, we can link it to the repository in `pingcap-incubator` org directly.